### DISCUSSION TITLE
[Feature] Add MFA Support

### DISCUSSION THREAD

**Discussion by athornfam2** [immich-app/immich#5478] [Feature Request]
### The feature

Please add native MFA/2FA support for non-Oauth accounts.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
When using oauth, MFA would be handled by whatever oauth server you're using.

**Reply by athornfam2** [reply]
I seemed to have missed "non" in my ask. I've updated the request to say non-oauth accounts. IE for those that don't want to setup a third party auth app.

**Comment by jrasm91** [comment]
We don't have plans to build a complicated auth system in immich. If you want that feature use a dedicated auth system and integrate it with immich via oauth.

**Comment by spupuz** [comment]
Would Be cool to have it natively
